src/TrashView.cpp

```diff
@@ -67,7 +67,7 @@ void CTrashView::AttachedToWindow( void )
 	if( watch_node( &nref, B_WATCH_DIRECTORY, BMessenger( this ) ) != B_NO_ERROR )
 		(new BAlert( "zz", "62 watch_node(  ) for Trash failed", "OK"  ))->Go();    
 		
-	menu = new BPopUpMenu( "HUH?" );
+	menu = new BPopUpMenu( "HUH?" ,false, false);
```

Comment:
I know, the code style of 8dock looks weird... :), stil:

```suggestion
	menu = new BPopUpMenu( "HUH?" , false, false);
```

Reply:
Done. 274b5e012113886c8d19087599489ec69516d862